solvers/julia_svrg.jl

```diff
@@ -0,0 +1,17 @@
+using StochOpt
+
+
+function solve_logreg_svrg(X, y, lambda, n_iter)
+    options = set_options(max_iter=n_iter, max_time=350.0, max_epocs=30,
+                          repeat_stepsize_calculation=true, rep_number=4,
+                          batchsize=100, embeddim=10, printiters=false);
+
+    probname = "empty"  # just a placeholder for the load logistic func below
+    prob = load_logistic_from_matrices(X, y, probname, options, lambda=lambda);
+    prob.fsol = 0.0  # disable loading of the solution
+    output = minimizeFunc(prob, "SVRG", options);
```

Comment:
Overall, I think that we could make sure we don't do unnecessary computation by not relying on `minizeFunc` but a simple script like:

```julia
function minimizeFunc(prob::Prob, method_input, options::MyOptions;)

    if options.initial_point == "randn" # set initial point
        x = randn(prob.numfeatures);
    elseif(options.initial_point == "rand")
        x = rand(prob.numfeatures); #
    elseif(options.initial_point == "ones")
        x = ones(prob.numfeatures); #
    else
        x = zeros(prob.numfeatures); #
    end

    if typeof(method_input) == String
        method = boot_method(method_input, prob, options);
        if method=="METHOD DOES NOT EXIST"
            println("FAIL: unknown method name:")
            return
        end
    else
        method = method_input
        method.bootmethod(prob, method, options)
    end

    d = zeros(prob.numfeatures); # Search direction vector

    for iter = 1:options.max_iter
        ## Taking a step
        method.stepmethod(x, prob, options, method, iter, d) # mutating function
        x[:] = x + method.stepsize * d
    end # End of For loop

    return x
end
```

Reply:
I'm not familiar with julia but this does not work if I define this function in the `julia_stochopt.jl` script -- because apparently all function define with `snake_case` (in this case `boot_method`) is not callable in this single script... Maybe an idea is to modify it from the source Stochopt package?

Reply:
arf....